lib/Core/PrettyExpressionBuilder.cpp

```diff
@@ -28,6 +28,9 @@ using namespace klee;
 
 namespace klee {
 
+const std::string array_prefix32 = "__arr32__";
+const std::string array_prefix64 = "__arr64__";
```

Comment:
@Himeshi It seems there are missing cases of arrays of 16 bits and 8 bits (byte) here.